src/analytics/ValoraAnalytics.ts

```diff
@@ -130,14 +131,8 @@ class ValoraAnalytics {
     }
 
     try {
-      const { walletAddress } = getCurrentUserTraits(store.getState())
-      const statsigUser =
-        typeof walletAddress === 'string'
-          ? {
-              userID: walletAddress,
-            }
-          : null
-
+      const defaultStatsigUser = getDefaultStatsigUser()
+      const statsigUser = defaultStatsigUser.userID ? defaultStatsigUser : null
```

Comment:
Should we set this regardless of the userID existing since there are other properties?

Reply:
I think it's safe to do so? I added this line in here for the sake of backwards compatibility, but I think we should be okay to set custom properties even when the user ID doesn't exist.